README.md

```diff
@@ -55,6 +55,11 @@ m5
     └── rconsortiumpilot3
         ├── tabulations
         |   ├── sdtm
+        |   │   ├── blankcrf.pdf
+        |   │   ├── define-v1-updated-html.xsl
+        |   │   ├── define.pdf
+        |   │   ├── define.xml
+        |   │   ├── pinnacle21-report-2023-05-19T10-33-03-557.xlsx
```

Comment:
Is there a way to remove and not copy the pinnacle21-report?  The FDA will most likely run this report on their end as well.  Usually P21 is for more of an 'internal' run prior to submitting to the FDA.

Best,
Joel

Reply:
Thanks @laxamanaj. The copy script shouldn't be copying that over, so maybe it was done manually at some point, but either way I've removed the P21 report and it shouldn't get copied over anymore.

Reply:
Thanks, @DeclanHodges